client/fns.go

```diff
@@ -67,11 +67,8 @@ func (n nonce) String() string {
 func (c *Cmd) UserKeyConfig() error {
 	kf := c.PrivateKeyFile
 	if len(kf) == 0 {
-		kf = config.Get(c.Host, "IdentityFile")
```

Comment:
This breaks all the ssh support from what I can see. Did I miss something?

Reply:
This change should not break any ssh use case if a user uses the `cpu` command binary. 

However, if a user used the `client` library and `c.PrivateKeyFile` is empty, yes the behavior is changed,
* before: `UserKeyConfig` will call `config.Get(c.Host, "IdentityFile")` to find a default key
* after: `UserKeyConfig` thinks that the user does not want to use a key.

If this kind of change is unwanted, what about adding an additional field `useKey` in the `Cmd` struct?

Reply:
sidecore uses that package and I'll need to see how this affects it. I think we should just add a useKey flag instead of counting on behavior when the string is zero length. My bad.

Reply:
I added a `DisablePrivateKey` field in the `Cmd` struct, and it is `false` as default. Then the behavior of the `client` package is unchanged.